tests/utils/concat-tests-test.js

```diff
@@ -16,8 +16,9 @@ var fixtureTestPath = path.join(__dirname, 'fixtures/packages/ember-metal/tests'
 
 describe('concat-es6', function() {
   var builder;
+  var packages = { 'ember-metal': {}, 'loader': {} };
```

Comment:
The changes made in this PR maintain existing behavior (I've tested by hand against `emberjs/ember.js`), but I'm not sure how to stub the tests to play nice with the files in `tests/fixtures`.

Any suggestions?

```
  1) concat-es6 correctly concats test tree into one file properly:
     Error: Path or pattern "**/*.js" did not match any files
    at Object.multiGlob (/Users/Sean/src/emberjs-build/node_modules/broccoli-static-compiler/node_modules/broccoli-kitchen-sink-helpers/index.js:202:13)
    at /Users/Sean/src/emberjs-build/node_modules/broccoli-static-compiler/index.js:25:27
    at $$$internal$$tryCatch (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:490:16)
    at $$$internal$$invokeCallback (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:502:17)
    at $$$internal$$publish (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:473:11)
    at Object.$$rsvp$asap$$flush [as _onImmediate] (/Users/Sean/src/emberjs-build/node_modules/broccoli/node_modules/rsvp/dist/rsvp.js:1581:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

cc: @twokul @rwjblue